src/Core__Type.res

```diff
@@ -30,13 +30,13 @@ type jsType =
   | Symbol(Core__Symbol.t)
   | Function(function)
 
-external toObjectUnsafe: 'a => object = "%identity"
-external toBoolUnsafe: 'a => bool = "%identity"
-external toFloatUnsafe: 'a => float = "%identity"
-external toBigIntUnsafe: 'a => Core__BigInt.t = "%identity"
-external toStringUnsafe: 'a => string = "%identity"
-external toSymbolUnsafe: 'a => Core__Symbol.t = "%identity"
-external toFunctionUnsafe: 'a => function = "%identity"
+let toObjectUnsafe = i => i->Obj.magic
+let toBoolUnsafe = i => i->Obj.magic
+let toFloatUnsafe = i => i->Obj.magic
+let toBigIntUnsafe = i => i->Obj.magic
+let toStringUnsafe = i => i->Obj.magic
+let toSymbolUnsafe = i => i->Obj.magic
+let toFunctionUnsafe = i => i->Obj.magic
```

Comment:
Sorry, this isn't what I meant. These are now all just aliases of `Obj.magic`, and wherever these are used internally you can just substitute with `Obj.magic` directly.

If you want to export them, then the externals are better because the implementation won't be hidden by the interface and consequently add a function call on every use to what should be a no-op at runtime.

Reply:
I see the problem. I looked at the javascript and there are all these wrappers and we want it to be completely compiled away. I went back to using external %identity in both the `resi` and the `res` because that results in less Js output. I find it very weird that the `resi` file has implementation details like `external`. If the `resi` just says `let toBoolUnsafe: 'a => bool` then this is the output...
```
function toBoolUnsafe(prim) {
  return prim;
}
```
In a perfect world wouldn't you be able to only use `external` in the implementation?